tests/common/test_checked.py

```diff
@@ -160,8 +161,9 @@ class Bar(Foo):
 
         tl = FooList([Bar(), Foo()])
         cls_name = self.BaseList.__name__
+        module_path = self.__module__
         expected = (
-            f"edb.common.checked.{cls_name}[common.test_checked."
+            f"edb.common.checked.{cls_name}[{module_path}."
```

Comment:
Btw, please use `self.assertEquals()` etc instead of the `assert` statement

Reply:
We used to use `pytest` at some point, then switched to a custom test running framework, and after that settled on using unittest. There are probably a few more tests that use the `assert` statement, all it says is that code is oooold.